brackend/schemagen.py

```diff
@@ -0,0 +1,165 @@
+def losersSameNextZeroJSON(winners, bracket):
+    #print("samenext zero")
+    for i in range(len(winners)-1):
+        for j in range(len(winners[i])):
+            if bracket["winners"][i]["seeds"][j]["teams"] != [{'name': '', 'game': -1}, {'name': '', 'game': -1}]:
+                if i == 0:
+                    winners[i][j][1][1] = 0
```

Comment:
you are referencing `winners[i][j]` a lot here. I am not sure what that contains. Can you give this a more descriptive name?